framework/src/schema/application_config_schema.ts

```diff
@@ -255,6 +255,11 @@ export const applicationConfigSchema = {
 					maximum: 30 * 1024, // Kilo Bytes
 					description: 'Maximum number of transactions allowed per block',
 				},
+				minimumCertifyHeight: {
+					type: 'integer',
+					minimum: 0,
```

Comment:
Will it work for minimum value zero? as we give default value as 1

Reply:
ah good point, i should change this to `1`